planetiler-core/src/test/java/com/onthegomap/planetiler/util/ParseTest.java

```diff
@@ -52,6 +53,39 @@ void testDirection(String in, int out) {
     assertEquals(out, Parse.direction(in));
   }
 
+  @ParameterizedTest
+  @CsvSource(value = {
+    "1,1",
+    "100,100",
+    "-1.23 m, -1.23",
+    "100.2,100.2",
+    "1m,1",
+    "1meter,1",
+    "100 meters,100",
+    "1.5m,1.5",
+    "1km,1000",
+    "0.2km,200",
+    "0.2 km,200",
+    "1mi,1609.344",
+    "1 mi,1609.344",
+    "328',99.974",
+    "328'11\",100.254",
+    "garbage,null",
+    "1nmi,1852",
+    "1.5 nmi,2778",
+    "1..5 nmi,null",
+    "36\",0.9144",
+    "1'11\",0.584",
```

Comment:
Another case to test would be decimal feet or inches, such as [`132.74'`](https://www.openstreetmap.org/node/9292162885) or `132'8.88"`.